Flutter/pdf-viewer/sticky-notes.md

```diff
@@ -0,0 +1,139 @@
+---
```

Comment:
Can we provide border or shadow for image?

<img width="567" alt="image" src="https://github.com/user-attachments/assets/a78172d7-6bc5-4033-aa73-0a65a5daac8d">